tests/type-check/main.arr

```diff
@@ -30,6 +31,13 @@ fun string-to-locator(name, str :: String):
   }
 end
 
+fun run-typed-file(base-path, filename) block:
+  file = FL.open-input-file(base-path + filename)
+  file-contents = FL.read-file(file)
+  FL.close-input-file(file)
+  TCH.run-to-result-typed(string-to-locator(filename, file-contents), base-path)
```

Comment:
This should be `base-path + filename`, right?

Reply:
Are you referring to the `filename` in the `string-to-locator` call?

Reply:
yes

Reply:
That shouldn't (and doesn't) make a difference. The first argument is just used for the `uri()` and `name()` in the locator.